integration_test/tests/chat/chat_group_test.dart

```diff
@@ -179,4 +180,25 @@ void main() {
 
     },
   );
+
+  TestBase().runPatrolTest(
+    description: 'View profile of a members in a group',
+    test: ($) async {
+      final s = SoftAssertHelper();
+      //goto chat
+      await HomeRobot($).gotoChatListScreen();
+
+      //open a group
+      const groupTest = String.fromEnvironment('GroupTest');
+      await ChatScenario($).openChatGroupByTitle(groupTest);
+
+      //open group Info
+      await ChatScenario($).openGroupChatInfo();
+
+      //Verify profile Info of all member
+      await ChatDetailScenario($).verifyProfileInfoOfAllMember(s);
+
+      s.verifyAll();
+    },
+  );
```

Comment:
```suggestion
  TestBase().runPatrolTest(
    description: 'View profile of a members in a group',
    test: ($) async {
      final s = SoftAssertHelper();
      await HomeRobot($).gotoChatListScreen();

      const groupTest = String.fromEnvironment('GroupTest');
      await ChatScenario($).openChatGroupByTitle(groupTest);

      await ChatScenario($).openGroupChatInfo();

      await ChatDetailScenario($).verifyProfileInfoOfAllMember(s);

      s.verifyAll();
    });
```

 -> 1/ Method names are exactly the comments, modulo camel case, the comments are thus redundant and can be stripped away. If that's unclear to you please take time to read `Clean Code: A Handbook of Agile Software Craftsmanship` 08/2008 from Robert Martin, it should come at a revelation...

 -> 2/ I hate lone closing brackets on lines and are in war with them :-)))

Reply:
Yes, in this case the method name is clear and matches my comments, so I’ll remove the comment.
But in cases where multiple lines are required to perform one behavior, I would like to add a comment to make it easier for the reader to understand